tools/score_instrument.php

```diff
@@ -145,41 +164,74 @@ function log_msg($message) {
 
         // check if the instrument has a scoring method
         if (!method_exists($instrument, "score")) {
-            fwrite(STDERR, "Error, the instrument ($test_name) does not have a scoring feature \n");
-            return false;
+            printOut("Error, the instrument ($test_name) does not have a scoring feature");
+            continue;
         }
 
         // print out candidate/session info
-        fwrite(STDERR, "Candidate: ".$record['CandID']."/".$record['Visit_label']."/".$record['SessionID'].":: (".$record['PSCID'].")\n");
-        //fwrite(STDERR, "Candidate: ".$instrument->_dob."/"."Test_name:".$test_name."/". $instrument->_pls3Age."/".$instrument->getDateOfAdministration().":: \n");
+        printOut("Candidate: ".$record['CandID']."/".$record['Visit_label']."/".$record['SessionID'].":: (".$record['PSCID'].")");
 
         // call the score function
-        $oldRecord = $db->pselectRow("SELECT * FROM $test_name WHERE CommentID=:cid", array('cid' => $record['CommentID']));
+        $oldRecord = $DB->pselectRow(
+            "SELECT * FROM $test_name WHERE CommentID=:cid",
```

Comment:
Should not select **\*** but specific field

Reply:
functionality change

Reply:
Right on that one, had not seen that the table is a variable that depend on the instrument.